src/test/java/org/isf/ward/test/Tests.java

```diff
@@ -148,11 +172,10 @@ public void testIoUpdateWard() {
 			foundWard.setDescription("Update");
 
 			// when:
-			boolean result = wardIoOperation.updateWard(foundWard);
+			wardIoOperation.updateWard(foundWard);
 			Ward updateWard = wardIoOperationRepository.findOne(code);
 
 			// then:
-			assertTrue(result);
```

Comment:
FWIW, by the fact that `assertTrue()` and `catch-try-fail` are present it appears that the `develop` branch this was based on is significantly out of date.

Reply:
@dbmalkovsky  sorry about that. I have updated my branch and re-merged the changes. Is this ok?

Reply:
There are still issues; see my comments in the source files.